assignment4.Rmd

```diff
@@ -115,4 +124,32 @@ biplot(pca)
 # Part III  
 
 ## Also in this repository is a data set collected from class that shows how many students thought that a TC program was related to andother TC program. Use PCA to look for components of related programs. Explain why you think there are relationships between these programs.
+```{r}
+T1 <- read.csv("TCprograms.csv", header = TRUE, sep="")
+T1 <- select(T1, -program)
+T1 <- scale(T1, center = TRUE)
+COR_T1 <- cor(T1)
+corrplot(COR_T1, order="AOE", method="circle", tl.pos="lt", type="upper",        
+tl.col="black", tl.cex=0.4, tl.srt=45, 
+        addCoef.col="black", addCoefasPercent = TRUE,
+        sig.level=0.50, insig = "blank")
+```
+```{r}
+pca_T1 <- prcomp(T1, scale = TRUE)
+pca_T1$sdev
+summary(pca_T1)
+plot(pca_T1, type = "lines")
+```
+```{r}
+T2 <- data.frame(pca_T1$x)
+COR_T2 <- cor(T2)
+corrplot(COR_T2, order="AOE", method="circle", tl.pos="lt", type="upper",        
+tl.col="black", tl.cex=0.6, tl.srt=45, 
+        addCoef.col="black", addCoefasPercent = TRUE,
+        sig.level=0.50, insig = "blank")
 
+loadings1 <- abs(pca_T1$rotation) 
+sweep(loadings1, 2, colSums(loadings), "/")
+biplot(pca_T1)
+```
+##The absolute value is near zero which means that the variable contributes little to the model.
```

Comment:
In general, good work.

> ##The absolute value is near zero which means that the variable contributes little to the model.

Not sure which value is near zero. But you could definitely look closely at the eigenvectors for each PC and interpret them in a more meaningful way.

> #Calculate values for each student that represent these your composite variables and then create a new correlogram showing their relationship to mean_correct.

Did you calculate this?